pkg/controllers/managementuser/nodesyncer/nodessyncer.go

```diff
@@ -822,6 +833,32 @@ func (m *nodesSyncer) isClusterRestoring() (bool, error) {
 		cluster.Spec.RancherKubernetesEngineConfig.Restore.Restore {
 		return true, nil
 	}
+	if cluster.Status.Driver == "imported" {
+		return false, nil
+	}
+	if strings.HasPrefix(cluster.Name, "c-m-") {
+		provCluster, err := m.provClusterCache.Get(cluster.Spec.FleetWorkspaceName, cluster.Spec.DisplayName)
```

Comment:
just a general comment but it's funny that we're using these fields... we should probably be setting these as an annotation on the v3 cluster object so we have a "solid" reference point.